docs/_docs/handling-exceptions.adoc

```diff
@@ -17,8 +17,120 @@
 
 This section outlines basic exceptions that can be generated by Ignite 3 and provides basic instructions for handling them.
 
-[cols="20%,40%,40%", width="100%"]
-|=======================================================================
-|Exception	|Description	|Action
+== Finding Stack Trace Information
 
-|`exception`|What causes exception|Recommended actions
\ No newline at end of file
+When the exception happens, Apache Ignite 3 provides a UUID of the specific exception, but not a full stack trace. For a full stack trace, check cluster logs.
+
+== Common Exceptions
+
+[cols="20%,80%", width="100%"]
+|===
+|Exception	|Description
+|`IGN-CMN-1`|Unexpected error occurred.
+|`IGN-CMN-2`|Operation was stopped because node is stopping.
+|`IGN-CMN-3`|Required component was not started.
+|`IGN-CMN-0xFFFF`|Deprecated. Unknown error occurred. Replaced by `IGN-CMN-1`.
```

Comment:
I don't think that we even need this in documentation. Anyway, this would probably lead to `IGN-CMN-65535`, because we render them as decimals